library/ssl_srv.c

```diff
@@ -3407,6 +3599,8 @@ static int ssl_parse_encrypted_pms( mbedtls_ssl_context *ssl,
 #pragma warning( pop )
 #endif
 
+    /* Set pms to either the true or the fake PMS, without
+     * data-dependent branches. */
```

Comment:
When using async, are there any timing differences between the use of real and fake PMS (maybe fake PMS doesn't go out to hardware, and so has different timing characteristics)?

Reply:
If both fake and true PMS are used equally by async, async is not a factor in introducing any additional side channels.

PMS aside, I'm curious what consideration has been given in the design of async as to avoiding introducing new potential side channels.